### DISCUSSION TITLE
Connecting to MS SQL server with domain auth

### DISCUSSION THREAD

**Discussion by walleXD** [prisma/prisma#18355] [Q&A] [answered]
Hey all,

I am trying to connect to a MS SQL server with prisma but have not been successful so far. How do I pass in the domain into the connection string? Usually, it's `DOMAIN\USERNAME` or passing the the domain explicitly while connecting from a SQL client like Datagrip. When I pass in user as `DOMAIN\USERNAME` it complains that the connection string is invalid.
<img width="1309" alt="Screenshot 2023-03-16 at 6 10 53 PM" src="https://user-images.githubusercontent.com/6826840/225763978-85e364bf-96f7-495f-92a5-872b271d7008.png">

I have tried to escape the `\` with `\\` but that doesn't work either. So, `DOMAIN\\USERNAME` still throws the same error.

Appreciate y'all help! 

Edit: I tried this approach too and it did not work https://github.com/prisma/prisma/issues/17056

**Comment by nurul3101** [comment, marked as answer]
Hello @walleXD 👋 

I was able to find a related GitHub Issue:
- https://github.com/prisma/prisma/issues/17056

Can you please try out the solution mentioned in this [comment](https://github.com/prisma/prisma/issues/17056#issuecomment-1396976391) and check if that works for you?

**Reply by walleXD** [reply]
I tried it. Getting the following error:

```
✖ Introspecting based on datasource defined in prisma/schema.prisma                                                                                                                                                                                               

Error: Login failed for user USER@DOMAIN
```

**Reply by walleXD** [reply]
Prisma is not using domain login and threating the entire username as user and not using the `@DOMAIN` part

**Reply by nurul3101** [reply]
I would recommend opening a [Bug Report](https://github.com/prisma/prisma/issues/new?assignees=&labels=kind%2Fbug&template=bug_report.yml) so that our engineering team can have a look and share their insights.
